weaver/datatype.py

```diff
@@ -821,7 +821,16 @@ def version(self):
     @property
     def inputs(self):
         # type: () -> Optional[List[Dict[str, Any]]]
-        return self.get("inputs")
+        inputs = self.get("inputs")
+        if inputs is not None:
+            for input_ in inputs:
+                maxOccurs = input_.get("maxOccurs", False)
+                minOccurs = input_.get("minOccurs", False)
+                if minOccurs:
+                    input_["minOccurs"] = int(minOccurs)
+                if maxOccurs and maxOccurs != "unbounded":
+                    input_["maxOccurs"] = int(maxOccurs)
```

Comment:
`minOccurs` and `maxOccurs` are mandatory in description

Were checks added because they were missing in come cases? If so, they should use the appropriate defaults, but I would tend to believe those defaults are added during deployment when omitted.

Reply:
Good point, check are not necessary there.